resources/latest/charts/cni/templates/zzz_profile.yaml

```diff
@@ -33,6 +33,13 @@ Finally, we can set all of that under .Values so the chart behaves without aware
 {{ fail (cat "unknown compatibility version" $.Values.compatibilityVersion) }}
 {{- end }}
 {{- end }}
+{{- if  $globals.platform }}
```

Comment:
@FilipB this is the change that is causing the issue, because `profile-openshift` does not exist anymore. I'm checking, why is only failing on that job